src/modules/utils.ts

```diff
@@ -60,6 +60,22 @@ const cleanVersion = (appId: string) => {
   )(appId)
 }
 
+// Return version tag
+// Example: 2.115.0-beta.somehash   -> beta
+// Example: 2.115.0                 -> latest
+export const getDistTag = (version: string) => {
+  const distTag = version.split('-')[1]
+  return distTag ? distTag.split('.')[0] : 'latest'
+}
+
+// Return version and tag only
+// Example: 2.115.0-beta.somehash   -> 2.115.0-beta
+// Example: 2.115.0                 -> 2.115.0
+export const getSimpleVersion = (_version: string) => {
+  const [version, distTag] = _version.split('-')
+  return version.concat(distTag ? '-'.concat(distTag.split('.')[0]) : '')
+}
```

Comment:
_suggestion_ (non blockin)

You can also use Regex to perform this operation
`/([0-9]+\.[0-9]+\.[0-9]+)/g`

You can use [the `match` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)

Reply:
Excellent idea!